### DISCUSSION TITLE
Do not use built-in or reserved HTML elements as component id: head

### DISCUSSION THREAD

**Discussion by 983796605** [nuxt/nuxt#9599] [Questions (Nuxt 2)] [answered]
error: Do not use built-in or reserved HTML elements as component id: head。
How to solve it

**Comment by kissu** [comment]
Hey 👋🏼, the same as on the previous question. 😄 

> Hey, could you please give us some more context and some code? Pretty hard to help here.

**Reply by 983796605** [reply]
> Hey 👋🏼, the same as on the previous question. 😄
> 
> > Hey, could you please give us some more context and some code? Pretty hard to help here.

I don't know. How did this error come from

**Reply by kissu** [reply]
Can't help without the context. The error is pretty explicit tho!

**Reply by 983796605** [reply]
> Can't help without the context. The error is pretty explicit tho!



> Can't help without the context. The error is pretty explicit tho!

ok,Thank you!

**Comment by TheAlexLichter** [comment, marked as answer]
Possibly you named a component `Head`. Use sth. like `MyHead` or `AppHead` instead because Vue components can't have names that are used by native HTML tags (like header, footer, head, ...)

**Reply by 983796605** [reply]
> Possibly you named a component `Head`. Use sth. like `MyHead` or `AppHead` instead because Vue components can't have names that are used by native HTML tags (like header, footer, head, ...)

ok,Thank you!

**Reply by 983796605** [reply]
> Possibly you named a component `Head`. Use sth. like `MyHead` or `AppHead` instead because Vue components can't have names that are used by native HTML tags (like header, footer, head, ...)

Excuse me. The reason is layout: "head" or Component head？

**Reply by kissu** [reply]
The name of the component. Maybe.
Can't be sure without any code.

**Reply by 983796605** [reply]
Thank you.
